ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Stages/FSharpQuickDocSuppressor.fs

```diff
@@ -0,0 +1,33 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Stages
+
+open JetBrains.DocumentModel.DataContext
+open JetBrains.ProjectModel
+open JetBrains.RdBackend.Common.Features.Daemon.Tooltips
+open JetBrains.RdBackend.Common.Features.Daemon.Tooltips.Request
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Psi.DataContext
+open JetBrains.Rider.Backend.Features.QuickDoc.Impl
+open JetBrains.TextControl
+open JetBrains.TextControl.DataContext
+
+[<SolutionComponent>]
+type FSharpQuickDocSuppressor
+    (
+        request: FSharpSyntaxHighlightingQuickDocTooltipRequest,
+        quickDocHighlighterManager: RiderQuickDocHighlighterManager
+    ) =
+    inherit QuickDocSuppressorBase()
+    with
+        override x.ShouldSuppressDocumentation context =
```

Comment:
@kirillgla Could you format it something like the following?
```fsharp
type FSharpQuickDocSuppressor(request: FSharpSyntaxHighlightingQuickDocTooltipRequest,
        quickDocHighlighterManager: RiderQuickDocHighlighterManager) =
    inherit QuickDocSuppressorBase()

    override this.ShouldSuppressDocumentation context = // no additional indent here, since redundant `with` is not added
        ...
```

It's surely not the most common way to format F# code, but I'm trying keep it consistent in the project anyway.